libmscore/chord.cpp

```diff
@@ -2376,7 +2376,7 @@ void Chord::layoutTablature()
       // allocate enough room for glissandi
       if (_endsGlissando) {
             if (!rtick().isZero())                        // if not at beginning of measure
-                  lll += (0.5 + score()->styleS(Sid::MinTieLength).val()) * _spatium;
+                  lll += _spatium * 0.5 + minTieLength;
```

Comment:
This is the same **formula** as for pitcher stafes, but not the same **value**, as for pitched staves `minTieLength` gets multplied with `mag_`, which is `staff() ? staff()->mag(this) : 1.0`.
So further up, there a need to add/chahe to

      qreal mag_              = staff() ? staff()->mag(this) : 1.0;    // palette elements do not have a staff
      qreal dotNoteDistance   = score()->styleP(Sid::dotNoteDistance) * mag_;
      qreal minNoteDistance   = score()->styleP(Sid::minNoteDistance) * mag_;
      qreal minTieLength      = score()->styleP(Sid::MinTieLength)    * mag_;

Reply:
See #7877